packages/core/src/client.tsx

```diff
@@ -61,12 +61,12 @@ export const useAnalytics = (): ClientMethods => {
     return {};
   }
   return {
-    screen: (...args) => client.screen(...args),
-    track: (...args) => client.track(...args),
-    identify: (...args) => client.identify(...args),
-    flush: () => client.flush(),
-    group: (...args) => client.group(...args),
-    alias: (...args) => client.alias(...args),
-    reset: (...args) => client.reset(...args),
+    screen: async (...args) => client.screen(...args),
```

Comment:
Adding `async` without `await` has a limited sense. It only moves the execution of client.screen() after the end of the current synchronous code but it doesn't allow to fully control the order of multiple concurrent event calls.